e2e/libs/persistent_volume_claim/rest.py

```diff
@@ -0,0 +1,15 @@
+from persistent_volume_claim.base import Base
+
+from utility.utility import get_longhorn_client
+
+class Rest(Base):
+
+    def __init__(self):
+        self.longhorn_client = get_longhorn_client()
```

Comment:
Is pvc part of Longhorn that can be manipulate by Longhorn client? If not, maybe we don't need this Rest implementation.